_posts/2022-04-11-1.15-announcement.md

```diff
@@ -0,0 +1,375 @@
+Next to this change, many bugs have been fixed and improvements made to the casting 
+functionality, to make sure that you’re getting correct results. The old CAST behavior 
+can be restored via a configuration flag. The effort is very well illustrated in this 
+JIRA issue. See the Flink documentation for details. 
+
+
+### JSON functions
+
+JSON is one of the most popular data formats and SQL users increasingly need to build 
+and read these data structures.  Multiple 
+[JSON](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/dev/table/functions/systemfunctions/#json-functions)
+functions have been added to Flink SQL 
+according to the SQL 2016 standard. It allows users to inspect, create and modify JSON 
+strings using the Flink SQL dialect.
+
+
+## Community enablement
+
+Enabling people to build streaming data pipelines to solve their use cases: That’s 
+what the Apache Flink community wants to do. The community is well aware that a 
+technology like Apache Flink is never used on its own and will always be part of a 
+bigger architecture. It is important to operate well on clouds, it is important to 
+connect to other systems, it is important to make the programming languages Apache 
+Flink supports work. Here’s what has been done in that regard.
+
+
+### Cloud Interoperability
+
+There are users operating Flink deployments in a cloud infrastructure of various 
+cloud providers, there are also services that offer to manage Flink deployments for 
+users on them. With Flink 1.15 an recoverable writer for Google Cloud Storage has 
+been added. Tidying up the connectors was also in scope and there has been work 
+especially on connectors for the AWS ecosystem. (e.g. 
+[KDS](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/connectors/datastream/kinesis/), 
+[Firehose](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/connectors/datastream/firehose/))
+
+
+### ElasticSearch sink
+
+As said, there was significant work on Flink’s connector ecosystem; still we want 
+to mention the 
+[ElasticSearch sink](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/connectors/datastream/elasticsearch/)
+at this point, because it was implemented following 
+very closely the new connector interfaces offering asynchronous functionality as well 
+as end-to-end semantics. This sink will act as a role model going forward.
+
+
+### Scala-free flink
+
+There was already a lengthy 
+[blogpost](https://flink.apache.org/2022/02/22/scala-free.html) by Seth Wiesman 
+in February explaining the ins and outs of why Scala users can now use the Flink 
+Java API with any Scala version - including Scala 3. At the end it was continuing 
+on what was started in 1.14 - removing the Mesos integration -, Akka has been isolated, 
+the DataStream / DataSet Java API has been cleaned up and hid the Table API behind an 
+abstraction. There’s already a lot of traction in the community.
+
```

Comment:
```suggestion

### Repeatable Cleanup

In previous releases of Flink, cleaning up job-related artifacts was done only once which might have resulted in abandoned artifacts in case of an error. In this version, Flink will try to run the cleanup again to avoid leaving artifacts behind. This retry mechanism runs until it was successful, by default. Users can change this behavior by configuring the [repeatable cleanup options](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/deployment/config/#retryable-cleanup). Disabling the retry strategy will lead to Flink behaving like in previous releases.

There is still work in progress around cleaning up Checkpoints, which is covered by [FLINK-26606](https://issues.apache.org/jira/browse/FLINK-26606).
```
Just in case this was missed accidentally. I think, the repeatable cleanup should end up in the release blog post as well.